<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>introducirEContr</class>
 <widget class="QDialog" name="introducirEContr">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="imagenes">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>131</width>
     <height>80</height>
    </rect>
   </property>
   <widget class="QWidget" name="page"/>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_poli">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>131</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_hf">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>131</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QLabel" name="label_hl">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>131</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="libertad">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>95</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Libertad del
Controlador</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>223</width>
     <height>70</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Numerador:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nume"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Denominador:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="deno"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>240</y>
     <width>176</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QPushButton" name="cancelar">
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="aceptar">
      <property name="text">
       <string>Aceptar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>199</width>
     <height>56</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="c_poli">
        <property name="text">
         <string>C. Polinomios</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="fl">
        <property name="text">
         <string>Formato Libre</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="hf">
        <property name="text">
         <string>k = hfgain</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="lf">
        <property name="text">
         <string>k = hlgain</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>161</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>K: [</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="kInicio"/>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>,</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="kFinal"/>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>]</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
